R/pkg/R/mllib.R

```diff
@@ -1027,7 +1009,7 @@ setMethod("spark.gaussianMixture", signature(data = "SparkDataFrame", formula =
 #' @export
 #' @note summary(GaussianMixtureModel) since 2.1.0
 setMethod("summary", signature(object = "GaussianMixtureModel"),
-          function(object, ...) {
```

Comment:
It's in the generic defined in a different file generics.R
We have discussed this at length, this is unfortunately a compromise we need to take at this point. We are going to see some documentation tags spread a bit from now on.

Long story short, CRAN checks warn about undocumented parameter; we need to have the generic with `...` to match the base::summary signature; we shouldn't add `...` in the actual definition since we don't use it and having it weakens the function contract (which is discouraged).